packages/utils/src/loader/cdn-loader.ts

```diff
@@ -49,48 +49,33 @@ export class CDNLoader {
    * @param href The location of the SVG to load
    * @returns Promise of the SVG body
    */
-  private async loadContent (href: string): Promise<XMLHttpRequest> {
+  private async loadContent (href: string): Promise<Response> {
     try {
-      const response = await this.xmlRequest(href);
-      return response;
+      return await fetch(href);
     }
     catch (e) {
       // Failed response...
       this.responseCache.delete(href);
+      let errorMessage = '';
+      if (e instanceof Error) {
+        errorMessage = e.message;
+      }
+      else if (e instanceof Response) {
+        errorMessage = e.statusText;
+      }
       return Promise.resolve({
-        status: 0, response: 'Failed to make request', responseText: 'Failed to make request'
-      } as XMLHttpRequest);
+        status: 0,
+        statusText: errorMessage
+      } as Response);
```

Comment:
Why you resolve this error when load content is fail and where this error will be used?

Reply:
Agree. We should response to tell error  some where.